toolkit/tools/scheduler/scheduler.go

```diff
@@ -69,6 +70,7 @@ var (
 	rpmmacrosFile              = app.Flag("rpmmacros-file", "Optional file path to an rpmmacros file for rpmbuild to use.").ExistingFile()
 	buildAttempts              = app.Flag("build-attempts", "Sets the number of times to try building a package.").Default(defaultBuildAttempts).Int()
 	checkAttempts              = app.Flag("check-attempts", "Sets the minimum number of times to test a package if the tests fail.").Default(defaultCheckAttempts).Int()
+	maxCascadingRebuilds       = app.Flag("max-cascading-rebuilds", "Sets the maximum number of cascading dependency rebuilds caused by package being rebuilt (< 0 for unbounded).").Default(defaultMaxCascadingRebuilds).Int()
```

Comment:
Came to my mind right now: for consistency across the code, maybe we can just use `NodeFreshnessAbsoluteMax`. This way there's only one source of truth saying what's the maximum. The default value for `MAX_CASCADING_REBUILDS` could be left blank and `--max-cascading-rebuilds` would be set only if it has an actual value.

```suggestion
	maxCascadingRebuilds       = app.Flag("max-cascading-rebuilds", "Sets the maximum number of cascading dependency rebuilds caused by package being rebuilt (< 0 for unbounded).").Default(schedulerutils.NodeFreshnessAbsoluteMax).Int()
```